vendor.conf

```diff
@@ -31,29 +31,29 @@ github.com/Microsoft/go-winio                       v0.4.14
 github.com/Microsoft/hcsshim                        v0.8.9
 github.com/opencontainers/go-digest                 v1.0.0
 github.com/opencontainers/image-spec                v1.0.1
-github.com/opencontainers/runc                      v1.0.0-rc10
-github.com/opencontainers/runtime-spec              v1.0.2
+github.com/opencontainers/runc                      v1.0.0-rc91
+github.com/opencontainers/runtime-spec              237cc4f519e2e8f9b235bacccfa8ef5a84df2875
 github.com/pkg/errors                               v0.9.1
 github.com/prometheus/client_golang                 v1.3.0
 github.com/prometheus/client_model                  v0.1.0
 github.com/prometheus/common                        v0.7.0
 github.com/prometheus/procfs                        v0.0.8
-github.com/russross/blackfriday                     v1.5.2
 github.com/sirupsen/logrus                          v1.6.0
 github.com/syndtr/gocapability                      d98352740cb2c55f81556b63d4a1ec64c5a319c2
-github.com/urfave/cli                               v1.22.0
+github.com/urfave/cli                               v1.22.1
 go.etcd.io/bbolt                                    v1.3.5
 go.opencensus.io                                    v0.22.0
+golang.org/x/xerrors                                9bdfabe68543c54f90421aeb9a60ef8061b5b544
```

Comment:
I'm new to this package. From the description:

> This repository holds the transition packages for the new Go 1.13 error values.
> See golang.org/design/29934-error-values.

Looks like this is for the new `errors.Is()`, `errors.Unwrap()` functionality in Go 1.13. We should look where it's used and if it's actually needed, now that all _current_ go versions support that


Looks like it's used by https://github.com/cilium/ebpf; https://github.com/cilium/ebpf/commit/5d50e74ed36f0420ef5004e84ed2cdd3e739ad3f (proposed in https://github.com/cilium/ebpf/issues/38). 

Perhaps we should propose them to drop Go 1.12 (as it's EOL), and use the standard `errors` package.

Reply:
Opened https://github.com/cilium/ebpf/pull/116 to remove this dependency

Reply:
thanks for filing https://github.com/cilium/ebpf/pull/116 @thaJeztah !

Reply:
https://github.com/cilium/ebpf/pull/116 was accepted and merged, so we could potentially update that ebpf and not need the new dependency. @dims you may know if it's used in other places where it should be updated first?

hmm.. I see it's marked as a dependency of `cgroups`, which may be on an older version that doesn't have the `x/xerrors` dependency; https://github.com/containerd/containerd/blob/a6dd1f27d963801d8237d14b1caa61762a388628/vendor.conf#L56-L57

Reply:
yea, it's `cgroups` and `runc`. should be ok to just update both when we get a chance and be ahead of them for now i think. I'll update to newer ebpf in this PR shortly

Reply:
Opened https://github.com/opencontainers/runc/pull/2493 and https://github.com/containerd/cgroups/pull/169